docs/vsphere_ci/scripts/install-openssh.ps1

```diff
@@ -0,0 +1,58 @@
+# Powershell script to configure OpenSSH Server in Windows Server 1809 and later.
+#
+# USAGE
+#    ./install-openssh.ps1 
```

Comment:
The `param()` function [defaults to `authorized_keys`](https://github.com/openshift/windows-machine-config-operator/blob/b6d41cdb3da46a77dd78f1650a94d4fb66285f1e/docs/vsphere_ci/scripts/install-openssh.ps1#L15) as the file name value [as documented](https://github.com/openshift/windows-machine-config-operator/blob/b6d41cdb3da46a77dd78f1650a94d4fb66285f1e/docs/vsphere_ci/scripts/install-openssh.ps1#L9) in the script.

Reply:
That is not clear from the documentation as that seems to describe the `keyfile` option.

Reply:
It can be made a bit more clear. Does it create an `authorized_keys` file in the current working directory? Or in the Admin's users home directory? Or in the `__PROGRAMDATA__/ssh/administrators_authorized_keys ` file? The script should attempt to do "the best practice" when no params is set.

Reply:
Good point, will adjust.